contributor-docs/discussion-docs/2016.md

```diff
@@ -0,0 +1,55 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# List Of Documents Submitted To dev@beam.apache.org In 2016
+| No. | Author | Subject | Date (UTC) |
+|---|---|---|---|
+| 1 | Jean-Baptiste | [Apache Beam: Technical Vision](https://docs.google.com/document/d/1UyAeugHxZmVlQ5cEWo_eOPgXNQA1oD-rGooWOSwAqh8/edit?pref=2&pli=1#heading=h.e5s64nliyukh) | 2016-02-09 11:01:41 |
+| 2 | Amit Sela | [Apache Beam (incubating) - Spark Runner Technical Vision](https://docs.google.com/document/d/1y4qlQinjjrusGWlgq-mYmbxRW2z7-_X5Xax-GG0YsC0/edit?usp=sharing) | 2016-02-10 13:46:37 |
+| 3 | Seetharam Venkatesh | [Issue Navigator - ASF JIRA](http://s.apache.org/gsoc2016ideas) | 2016-02-15 13:38:18 |
+| 4 | Kenneth Knowles | [Lateness (and Panes) in Apache Beam (incubating)](https://docs.google.com/document/d/12r7frmxNickxB5tbpuEh_n35_IJeVZn1peOrBrhhP6Y/edit?usp=sharing) | 2016-03-01 11:50:30 |
+| 5 | Tyler Akidau | [Beam Model & Runner Compatibility Matrix - Google Sheets](https://docs.google.com/spreadsheets/d/1OM077lZBARrtUi6g0X0O0PHaIbFKCD6v0djRefQRE1I/edit) | 2016-03-09 17:14:29 |
+| 6 | Ben Chambers | [Static Display Data](https://docs.google.com/document/d/11enEB9JwVp6vO0uOYYTMYTGkr3TdNfELwWqoiUg5ZxM/edit?usp=sharing) | 2016-03-14 20:25:44 |
+| 7 | Kenneth Knowles | [Apache Beam (incubating) Pipeline Runner API 1-Pager](https://docs.google.com/document/d/1zEJN06YaKEIMhSckVjVgLQIh-jxqkgDRSnos1AOVSQ4/edit?usp=sharing) | 2016-03-15 11:28:45 |
+| 8 | Frances Perry | [[Draft] Apache Beam Contribution Guide](https://docs.google.com/document/d/1syFyfqIsGOYDE_Hn3ZkRd8a6ylcc64Kud9YtrGHgU0E/comment) | 2016-03-17 17:19:08 |
+| 9 | Jean-Baptiste | [[Draft] Apache Beam Contribution Guide](https://docs.google.com/document/d/1syFyfqIsGOYDE_Hn3ZkRd8a6ylcc64Kud9YtrGHgU0E/) | 2016-03-21 12:59:42 |
```

Comment:
We might be able to dedupe entries like this by using som string prefix comparisons

Reply:
Updated all MD files by standardizing the urls.